src/Backend/Modules/Blog/Installer/Installer.php

```diff
@@ -144,18 +145,18 @@ public function install()
         $this->setNavigation($navigationModulesId, 'Blog', 'blog/settings');
 
         // add extra's
-        $blogId = $this->insertExtra('Blog', 'block', 'Blog', null, null, 'N', 1000);
-        $this->insertExtra('Blog', 'widget', 'RecentComments', 'RecentComments', null, 'N', 1001);
-        $this->insertExtra('Blog', 'widget', 'Categories', 'Categories', null, 'N', 1002);
-        $this->insertExtra('Blog', 'widget', 'Archive', 'Archive', null, 'N', 1003);
-        $this->insertExtra('Blog', 'widget', 'RecentArticlesFull', 'RecentArticlesFull', null, 'N', 1004);
-        $this->insertExtra('Blog', 'widget', 'RecentArticlesList', 'RecentArticlesList', null, 'N', 1005);
+        $blogId = $this->insertExtra('Blog', ExtraType::block(), 'Blog', null, null, 'N', 1000);
+        $this->insertExtra('Blog', ExtraType::widget(), 'RecentComments', 'RecentComments', null, 'N', 1001);
+        $this->insertExtra('Blog', ExtraType::widget(), 'Categories', 'Categories', null, 'N', 1002);
+        $this->insertExtra('Blog', ExtraType::widget(), 'Archive', 'Archive', null, 'N', 1003);
+        $this->insertExtra('Blog', ExtraType::widget(), 'RecentArticlesFull', 'RecentArticlesFull', null, 'N', 1004);
+        $this->insertExtra('Blog', ExtraType::widget(), 'RecentArticlesList', 'RecentArticlesList', null, 'N', 1005);
```

Comment:
That is correct. For now, the value object `ExtraType` with type `ExtraType::WIDGET` just get's casted back to `ExtraType::WIDGET` but in the future we can assert that an `ExtraType` value object should be sent instead of a string. Doing this will make sure we have a valid extra type in our database